### PR TITLE
T11- Digital root of a number.

### PR DESCRIPTION
**Pull request by Priyanka15-12**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter a number to find digit root: ");
+         int n = int.Parse (Console.ReadLine ()); 
+         int quo, rem ;
```
Comment (MetaNaveen) [inline Program.cs:6]: space not required before semi colon.

File: Program.cs [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
+         while (n > 9) {
+            int add = 0;
+            while (n > 0) {
+               quo = n / 10;
+               rem = n % 10;
```
Comment (MetaNaveen) [inline Program.cs:11]: These 2 lines could be used directly in the next 2 lines.

**Review by MetaNaveen — CHANGES_REQUESTED** [review]

**Review by MetaNaveen — APPROVED** [review]

**Comment by MetaNaveen** [comment]
Working as expected.

**Final Notes:**
Should have prompted for "positive" numbers.

**Comment by Priyanka15-12** [comment]
Modified

**Pull request closed without merging**